playbooks/roles/plugins/openscap/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+openscap_hammer: hammer -u admin -p changeme
```

Comment:
This isn't too robust against the admin password getting configured to something else.